.gitignore

```diff
@@ -254,3 +254,6 @@ ModelManifest.xml
 /src/Stratis.StratisD/Logs
 /src/Stratis.BitcoinD/Logs
 /src/Stratis.Benchmark/BenchmarkDotNet.Artifacts/results
+/src/Stratis.FederatedSidechains.AdminDashboard/wwwroot/css/*.min.css
```

Comment:
I migrate some rules in the whole repo .gitignore

Reply:
Nice.